desc/verbose.txt

```diff
@@ -0,0 +1,41 @@
+__main__.py:    if ir.args.verbose:
```

Comment:
what is this file for?

Reply:
That's an intermediate output file I git grep'ed into; removing.  Wasn't quite finished last night waiting for unittests to come back; had to fix two other bugs like this I omitted as well.